Week_01_Basics/Assignment/Homework Submission/Stylesheet2.css

```diff
@@ -0,0 +1,18 @@
+body{
+	background-color:#F5F5F5;
+}
+nav{
+	text-align: center;
+	font-size: 20px;
+}
+h2{
+	text-align: center;
+	font-size: 35px
+	bold
+}
+#top{
+	border:1px solid black;
```

Comment:
Your code style here needs to be a little more consistent.  You need a space after the colon
```border: 1px solid black;```